system_metrics_collector/package.xml

```diff
@@ -16,6 +16,9 @@
   <build_depend>rcpputils</build_depend>
   <build_depend>metrics_statistics_msgs</build_depend>
 
+  <!--Required for example launch file-->
```

Comment:
Not sure these comments are necessary but sure.

Reply:
Just to clarify why we are depending on a demo project :-)